content/docs/04-clusters/03-edge/04-edgeforge-workflow.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "EdgeForge Workflow"
+metaTitle: "Edge Artifact Builder Workflow"
+metaDescription: "Learn how to build your own Edge artifacts customized to your specific needs."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview
+
+*EdgeForge* is the process or workflow of preparing an Edge host with all the required components and dependencies. The EdgeForge workflow contains several steps and key elements that you must complete to ensure the Edge host is ready for a successful site deployment. 
+
+EdgeForce contains three critical components.
+
+<br />
+
+* Edge Installer ISO.
+
+
+* Edge Host Agent Container Image.
+
+
+* Edge Provider Container Images.
+
+
+
+Each component plays a critical role in the [lifecycle](/clusters/edge/edge-native-lifecycle) of an Edge deployment. Review the following sections to learn more about each component.
+
+![A diagram that displays the relationship between the three components  and how they relate to an Edge host](/clusters_edge-forge-workflow_edgeforge-workflow_components-diagram.png)
+
+
+
+## Edge Installer ISO
+
+An ISO file that bootstraps the installation process. The ISO image contains the Edge Installer that installs the Palette Edge host agent and metadata to perform the initial installation. The default Edge Install ISO download URL is in the [Downloads](/spectro-downloads#edgeinstallimages) resource.
+
+<br />
+
+## Edge Host Agent Container Image
+
+The Edge host agent container image contains the Palette Edge host agent. The agent is responsible for Day-2 operations and management of the Edge host. The Edge host agent will also provide ongoing support during cluster runtime.
+
+<br />
+
+## Edge Provider Container Images
+
+[Kairos](https://kairos.io/)-based container images for each supported Operating System (OS) and Kubernetes combination. These container images are downloaded during the installation by the Edge Installer and converted to disk images for the system to boot into.
+
+Palette provides these artifacts out-of-the-box. All the container images are hosted in Palette's container registries and downloaded during installation. You can use the default Palette container registries to familiarize yourself with the installation process. However, in a typical production scenario, you would need to customize these artifacts to suit your specific needs or perform some [content bundle](/clusters/edge/edgeforge-workflow/build-content-bundle) optimization.
```

Comment:
"...downloaded during installation."  Does the user do the downloading or is there some auto-downloading process that happens?

Reply:
Good point. Adding some clarification.